packages/main/src/plugin/extension-loader.ts

```diff
@@ -293,12 +295,12 @@ export class ExtensionLoader {
 
     const analyzedFoldersExtension = (
       await Promise.all(folders.map(folder => this.analyzeExtension(folder, false)))
-    ).filter(extension => extension !== undefined) as AnalyzedExtension[];
+    ).filter(extension => !extension.error) as AnalyzedExtension[];
```

Comment:
Here and below `as AnalyzedExtension[]` is not needed.
```suggestion
    ).filter(extension => !extension.error) as AnalyzedExtension[];
```

Reply:
Fixed.